scss/_list-group.scss

```diff
@@ -132,18 +132,16 @@
 // useful within other components (e.g., cards).
 
 .list-group-flush {
-  .list-group-item {
+  > .list-group-item {
     border-right-width: 0;
     border-left-width: 0;
     @include border-radius(0);
 
     &:first-child {
       border-top-width: 0;
     }
-  }
 
-  &:last-child {
-    .list-group-item:last-child {
+    &:last-child {
```

Comment:
This breaks the kitchen sink example:
https://deploy-preview-30039--twbs-bootstrap.netlify.com/docs/4.3/components/card/#kitchen-sink